src/utils/tokenIds.ts

```diff
@@ -42,7 +42,7 @@ const isBaseType = (id: bigint) => {
  * @param {BigInt} tokenId - The token ID to check. It can be undefined.
  * @returns {boolean} - Returns true if the token ID represents a claim token, false otherwise.
  */
-export const isClaimToken = (tokenId?: bigint) => {
+export const isHypercertToken = (tokenId?: bigint) => {
```

Comment:
i feel like renaming this to hypercert token makes sense, but what does that mean for all the other `Claim`-related naming in our codebase?

Reply:
Would be a nice refactor as we're learning hypercert & fraction is clearer than claim & fraction.